packages/homepage/content/articles/templates.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
@Saeris You can make move this to it's own `templates` folder within the `articles` folder an let it have it's own  `images` folder inside it.